addon/system/store/finders.js

```diff
@@ -56,6 +60,7 @@ export function _findMany(adapter, store, typeClass, ids, internalModels) {
   promise = _guard(promise, _bind(_objectIsAlive, store));
 
   return promise.then(function(adapterPayload) {
+    Ember.assert("You made a `findMany` request for a " + typeClass.typeClassKey + " with ids " + ids + ", but the adapter's response did not have any data", payloadIsNotBlank(adapterPayload));
```

Comment:
@bmac unfortunately, there isn't a `store` operation that maps directly to this one like the others.

For instance, `adapter.findRecord` maps to `store.find`, `adapter.query` maps to `store.query` (and maybe `store.queryRecord`?).

In this case, isn't `findMany` used to coalesce find requests?

Would I be able to exercise `adapter.findMany` with a `store.query({ ids: [1,2,3] })`?